stress/src/throughput.rs

```diff
@@ -28,7 +29,28 @@ where
         STOP.store(true, Ordering::SeqCst);
     })
     .expect("Error setting Ctrl-C handler");
-    let num_threads = num_cpus::get();
+
+    let args: Vec<String> = env::args().collect();
```

Comment:
nit: You could the `nth` method to get the first command line argument instead of collecting the arguments into a new vector.

```rust
if let Some(num_threads_arg) = env::args().nth(1) {
    // validate the command line argument
}
```

Reply:
We should probably look at using some crate like [clap](https://docs.rs/clap/latest/clap/) to support named command line arguments.